From b2067f90336f7af41b5468fb7ca5e187a26b4576 Mon Sep 17 00:00:00 2001
From: "Thulasimani,Sivakumar" <sivakumar.thulasimani@intel.com>
Date: Wed, 23 Mar 2016 11:15:49 +0530
Subject: [PATCH 04/19] Revert "FOR_UPSTREAM [VPG]: drm/i915: Avoid optimizing
 lane count"

This reverts commit a5377478c8bc142710cd1af64eb6874e4855b2e9.
It is not needed anymore since we have a patch that
avoids lane count optimization for external panels instead
of current patch which does so only for compliance test
cases.

Change-Id: Id65e5e26e583a78dae20f7b4e3ed05ca179a469a
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-30099
Signed-off-by: Sivakumar Thulasimani <sivakumar.thulasimani@intel.com>
Reviewed-on: https://android.intel.com:443/498179
---
 drivers/gpu/drm/i915/intel_dp.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp.c b/drivers/gpu/drm/i915/intel_dp.c
index a56e11d..1a516d1 100644
--- a/drivers/gpu/drm/i915/intel_dp.c
+++ b/drivers/gpu/drm/i915/intel_dp.c
@@ -1114,13 +1114,6 @@ found:
 	if (intel_dp->color_range)
 		pipe_config->limited_color_range = true;
 
-	/*
-	 * compliance tests expect same lane count as reported, so
-	 * avoid any optimization during tests
-	 */
-	if (intel_dp->compliance_test_type == DP_TEST_LINK_TRAINING)
-		lane_count = max_lane_count;
-
 	intel_dp->link_bw = bws[clock];
 	intel_dp->lane_count = lane_count;
 	pipe_config->pipe_bpp = bpp;
-- 
1.9.1

